docker/py/Dockerfile

```diff
@@ -42,7 +41,7 @@ RUN groupadd -g 1000 jovyan
 USER $NB_USER
 
 # install jupyterlab, papermill, git extension and renku-jupyterlab-ts
-COPY requirements.txt /tmp/requirements.txt
+COPY docker/py/requirements.txt /tmp/requirements.txt
```

Comment:
@rokroskar why did you change the context for the builds above?

I.e. we had `docker build <path to docker image folder>` but you switched it to `docker build . -f <full_path_to_dockerfile>`.

I think it makes more sense for the build context to be the location of each specific `Dockerfile` and not the root of the repository.

Also with this change now we have some images whose build contexts are in the same folder as the Dockerfile and some where the build context is `.` (i.e. the root of the repo).

I think it is best to keep this consistent and to keep things as they were. The build context for each image is the folder where the Dockerfile lives and not the root of the repository.

Reply:
yeah this is a bit dirty, I agree, and I don't really like it. But I want to have one location for the renku dependency for any of the images that need it.

Reply:
The alternative is to have the requirements file in multiple places. Not a horrible solution but not a great one.

Reply:
@olevski I went back to the previous structure. Please have a look 🙏